docs/release_process.md

```diff
@@ -12,52 +11,120 @@ Everyone is welcome to test the stabilization branch and propose fixes.
 This process reduces the risk of bringing a sudden breaking change into the release,
 and ultimately allows the release process to happen while development continues on master branch.
 
-To start the stabilization:
-
-- Create a new branch called **stabilization-vX.Y.Z**, e.g.: stabilization-v0.1.61
-
-- Create a new milestone for the next version
-
-- Update the milestone of open pull requests to the new one, if they have any
-
-- Update the milestone of open issues to the new one, if they have any
-
-- Close the milestone for the current release.
-  This makes the milestone less visible and reduces the chance of PRs and Issues being added to it
-  accidentally.
-  It is still possible to add items to the closed Milestone, just select it in the `closed` tab.
-
-- Bump the version of `CMakeLists.txt` on the **master** branch.
-
-- Announce the start of stabilization period on the mailing list
-
-During the stabilization:
-
-- Run whatever extra tests you have and propose bug fixes to the stabilization branch
-
-- On Fridays, bring the fixes merged on **stabilization** branch to master.\
-  Propose a PR from **stabilization** branch to master.
+## Preparing the stabilization
+
+- Create a new branch called **stabilization-vX.Y.Z** where **X.Y.Z** is the version of upstream
+release to be created.
+    - For example, if the last released version was **0.1.64**, the branch should be named as
+    `stabilization-v0.1.65`
+<!-- -->
+- Create a new Milestone called **X.Y.Z**, where **X.Y.Z** is the version of future upstream release.
+    - For example, if you just created the `stabilization-v0.1.65` branch in order to start the
+    stabilization phase for **0.1.65**, the new Milestone should be `0.1.66`.
+<!-- -->
+- All Pull Requests which are open at the moment the stabilization branch was created won't be
+included in the respective release. Therefore, if they have a Milestone defined, the Milestone
```

Comment:
```suggestion
included in this release. Therefore, if they have a Milestone defined, they
```

Reply:
Why is it better to change `the respective` by `this`? Could you share how did you interpret it, please?

Reply:
To me, `respective` gives the signal that we are handling multiple releases, that each PR has a release pertaining to it. Simplifying and reducing the sentence: `Pull Requests won't be included in the respective release`. And it should probably be `their respective`. 
Ref: https://www.collinsdictionary.com/us/dictionary/english/respective

Instead of `this`, you could also use `current`.

Reply:
Thanks for sharing your thoughts. So, changing to `current` makes more sense to me in order to reduce the risk of misinterpretation. Anyway, I will review how to improve the complete sentence.

Reply:
I updated the sentence. Could you take a look again, please?

Reply:
Looks good.